_layouts/bib.liquid

```diff
@@ -261,12 +272,14 @@
               data-hide-less-than="15"
               data-badge-type="2"
               data-badge-popover="right"
-              {% if entry.eprint %}
+              {% if entry.altmetric != blank and entry.altmetric != 'true' %}
```

Comment:
Shouldn't it be `! = false` here?

Reply:
Not really, I think the expected values here are `"true"`, `"false"`, and any altmetric id string. `"true"` means that the user wants to enable the badge. We only want to enter this condition check when the value of the altmetric entry is set to an id, not set to `"true"`.

Reply:
Oh if altmetric is `"false"`, then `entry_has_altmetric_badge` won't be set to `true` in [line 254](https://github.com/alshedivat/al-folio/pull/2522/commits/d8917e7836389f21ed471b9f175fafd913ad148b#diff-4ef4ea42cd30776daef2d2cc2da3e79af05331b84aa21bc8e7eea019b935d701R253-R254) and the code wont pass the condition check in [line 268](https://github.com/alshedivat/al-folio/pull/2522/commits/d8917e7836389f21ed471b9f175fafd913ad148b#diff-4ef4ea42cd30776daef2d2cc2da3e79af05331b84aa21bc8e7eea019b935d701R268). So we don't need to worry about "false" in [line 275](https://github.com/alshedivat/al-folio/pull/2522/commits/d8917e7836389f21ed471b9f175fafd913ad148b#diff-4ef4ea42cd30776daef2d2cc2da3e79af05331b84aa21bc8e7eea019b935d701R275)